README.md

```diff
@@ -161,4 +164,4 @@ I would like to thank Microverse for the opportunity to dream big.
 This project is [GNsU](./LICENSE) licensed.
```

Comment:
- [ ] I would kindly suggest you work on the project license as the link is currently broken. with a functioning link you'd be sure you have a professional README file.

License

![mit](https://user-images.githubusercontent.com/78634841/233767734-9da295d0-053f-4732-94e8-da53d75c0bea.png)